packages/flutter/test/widgets/title_test.dart

```diff
@@ -14,4 +15,53 @@ void main() {
     );
     expect(widget.toString, isNot(throwsException));
   });
+
+  testWidgets('should handle having no title', (WidgetTester tester) async {
+    final Title widget = new Title(
+      child: new Container(),
+      color: const Color(0xFF00FF00),
+    );
+    expect(widget.toString, isNot(throwsException));
+    expect(widget.title, equals(''));
+    expect(widget.color, equals(const Color(0xFF00FF00)));
+  });
+
+  testWidgets('should not allow null title or color', (WidgetTester tester) async {
+    expect(() => new Title(
+      title: null,
+      color: const Color(0xFF00FF00),
+      child: new Container(),
+    ), throwsAssertionError);
+    expect(() => new Title(
+      color: null,
+      child: new Container(),
+    ), throwsAssertionError);
+  });
+
+  testWidgets('should not allow non-opaque color', (WidgetTester tester) async {
+    expect(() => new Title(
+      color: const Color(0),
+      child: new Container(),
+    ), throwsAssertionError);
+  });
+
+  testWidgets('should not pass "null" to setApplicationSwitcherDescription',
+          (WidgetTester tester) async {
+    final List<MethodCall> log = <MethodCall>[];
+
+    SystemChannels.platform.setMockMethodCallHandler((
+        MethodCall methodCall) async {
```

Comment:
wrapping rule is if you have a newline after a `(` there should beone before the matching `)`

Reply:
Done.